gulpfile.js

```diff
@@ -40,6 +41,18 @@ gulp.task('serve:dist', ['build:dist'], require('./tasks/server')(gulp, config.d
 gulp.task('serve', ['compile:src', 'less', 'fonts', 'html', 'json', 'assets'],
     require('./tasks/server')(gulp, config.srcDir, config.watchDir, true, config.projectDir, config));
 
+gulp.task('typedoc', function() {
+  return gulp
+    .src(['src/services/**/*.ts'])
+    .pipe(typedoc({
+      module: 'commonjs',
+      target: 'es5',
+      out: 'docs/',
+      name: 'Infinispan management console'
+    }))
+    ;
+});
+
```

Comment:
This method should be in its own file like the other gulp tasks. `gulpfile.js` is just an entry point.